docs/extend/modular-resources/key-concepts.md

```diff
@@ -95,9 +95,11 @@ If you are [creating a custom module](/extend/modular-resources/create/) and [up
 - Your model triplet must be all-lowercase.
 - Your model triplet may only use alphanumeric (`a-z` and `0-9`), hyphen (`-`), and underscore (`_`) characters.
 
-In addition, you should chose a name for the `family` of your model based on the whether your module implements a single model, or multiple models:
+In addition, you should use the name of the git repository for your modular resource as the `repo-name`.
+The `repo-name` should identify a single model or multiple models.
+You can use a name that identifies the _family_ of the model or models:
 
-- If your module provides a single model, the `family` should match the `subtype` of whichever API your model implements.
+- If your module provides a single model, the `repo-name` should match the `subtype` of whichever API your model implements.
```

Comment:
I think that when Eliot said "repo-name", he literally meant the name of the github repository. So in case of intel realsense, the repo name is "viam-camera-realsense": https://github.com/viamrobotics/viam-camera-realsense


We should ask Eliot in product sync if he wants us to change any repo names (should be pretty easy lift) in order to have good examples to document.